tests/pyqtslot.py

```diff
@@ -1,7 +1,12 @@
 from PyQt5.QtCore import pyqtSlot
 
 @pyqtSlot(str)
-def func(s: str) -> int:
+def func_none(s: str) -> None:
```

Comment:
Do we want to add some tests that force/catch an exception?

Reply:
What exceptions do you have in mind?  Or do you mean type hint failures?

Reply:
Yeah. Tests cases that raise mypy errors to demonstrate that the annotations are correct.

Reply:
I didn't expect to get into that until a better testing setup is worked on.  Perhaps what is mentioned in #11.  I haven't dug much into that yet.